extensions/sql-migration/src/constants/strings.ts

```diff
@@ -352,18 +353,31 @@ export const STARTING_LOGIN_MIGRATION = localize('sql.migration.starting.login',
 export const ESTABLISHING_USER_MAPPINGS_FAILED = localize('sql.login.migration.establish.user.mappings.failed', "Establishing user mappings failed");
-export const MIGRATE_SERVER_ROLES_AND_SET_PERMISSIONS = localize('sql.login.migration.migrate.server.roles.and.set.permissions', "Establishing user mappings completed.\n\nCurrently, migrating server roles, establishing server mappings and setting permissions. This will take some time.");
-export const MIGRATE_SERVER_ROLES_AND_SET_PERMISSIONS_FAILED = localize('sql.login.migration.migrate.server.roles.and.set.permissions.failed', "Migrating server roles, establishing server mappings and setting permissions failed.");
+export const MIGRATING_SERVER_ROLES_AND_SET_PERMISSIONS = localize('sql.login.migration.migrate.server.roles.and.set.permissions', "Establishing user mappings completed.\n\nCurrently, migrating server roles, establishing server mappings and setting permissions. This will take some time.");
+export const MIGRATING_SERVER_ROLES_AND_SET_PERMISSIONS_FAILED = localize('sql.login.migration.migrate.server.roles.and.set.permissions.failed', "Migrating server roles, establishing server mappings and setting permissions failed.");
 export const LOGIN_MIGRATIONS_COMPLETE = localize('sql.login.migration.complete', "Completed migrating logins");
 export const LOGIN_MIGRATIONS_FAILED = localize('sql.login.migration.failed', "Migrating logins failed");
 export function LOGIN_MIGRATIONS_ERROR(message: string): string {
-	return localize('sql.login.migration..error', "Login migration error: {0}", message);
+	return localize('sql.login.migration.error', "Login migration error: {0}", message);
 }
 export const LOGINS_FOUND = localize('sql.login.migration.logins.found', "Login found");
 export const LOGINS_NOT_FOUND = localize('sql.login.migration.logins.not.found', "Login not found");
 export const LOGIN_MIGRATION_STATUS_SUCCEEDED = localize('sql.login.migration.status.succeeded', "Succeeded");
 export const LOGIN_MIGRATION_STATUS_FAILED = localize('sql.login.migration.status.failed', "Failed");
 export const LOGIN_MIGRATION_STATUS_IN_PROGRESS = localize('sql.login.migration.status.in.progress', "In progress");
+export const LOGIN_MIGRATIONS_AAD_DOMAIN_NAME_INPUT_BOX_LABEL = localize('sql.login.migration.aad.domain.name.input.box.label', "Azure Active Directory Domain Name (only required to migrate Windows Authenication Logins)");
+export const LOGIN_MIGRATIONS_AAD_DOMAIN_NAME_INPUT_BOX_PLACEHOLDER = localize('sql.login.migration.aad.domain.name.input.box.placeholder', "Enter AAD Domain Name");
+export function LOGIN_MIGRATIONS_LOGIN_STATUS_DETAILS_TITLE(loginName: string): string {
+	return localize('sql.login.migration.login.status.details.title', "Migration status details for {0}", loginName);
+}
+export const LOGIN_TYPE_WINDOWS_LOGIN = localize('sql.login.type.windows.login', "WINDOWS_LOGIN");
```

Comment:
presumably this string "WINDOWS_LOGIN" isn't intended to be shown to the user - if so I think this should live as a normal const wherever we're using it, instead of having it here - I believe this is only meant for localizable strings

Reply:
ah gotcha. yeah, this is used to check what type of login it.

I understand your point that this file is for localizable strings. Seems like your suggestion is use a local const. Would it make sense to another section in consts or somewhere else to define consts like this for comparision?

Reply:
moved to enum!